cmd/varint/flag.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
This whole file is borrowed from cmd/hex. Should we move this stuff somewhere where both commands can use it?